R/intensity_levels.R

```diff
@@ -0,0 +1,77 @@
+#' Compute intensity levels with seasonal time series observations.
+#'
+#' @description
+#' `r lifecycle::badge("stable")`
+#'
+#' This function calculates the intensity levels of time series observations that are stratified by season.
+#' It uses the previous seasons to calculate the intensity levels of the newest season.
+#' It can provide two different types of intensity levels, by using one of following functions:
+#' - `compute_weighted_intensity_levels` which calculates the lower, medium and high intensity levels.
+#' - `compute_relative_dist_intensity_levels` which calculates the very low, low, medium and high
```

Comment:
Since `compute_relative_dist_intensity_levels` is a function to come in the future, I cannot comment on what it will do, but
looking at the two function names, I suggest maybe renaming to make them easier to understand.

Both will use the backward reduced weigting of old seasons?
If that is the case, I suggest naming something like:

`compute_weighted_intensity_levels()` -> `compute_absolute_intensity_levels()`
`compute_relative_dist_intensity_levels()` -> `compute_relative_intensity_levels()`

Reply:
This is mostly a suggestion for you to discuss with Lasse and implement when you make the "compute_relative_dist_intensity_levels" function